Zotero/Extensions/NSUserActivity+Activities.swift

```diff
@@ -11,6 +11,7 @@ import Foundation
 struct RestoredStateData {
     let key: String
     let libraryId: LibraryIdentifier
+    let collectionId: CollectionIdentifier?
```

Comment:
Should `CollectionIdentifier` here be optional? I know it's not stored yet, but we could default to all items or `Defaults.shared.selectedCollectionId` for first-time use. Otherwise library and collection should always be available?

Reply:
If we have a default option as fallback, we could do that. I'll check existance in order,
user info value > `Defaults.shared.selectedCollectionId` > all items